sdk/orm-django/django_encryption/fields.py

```diff
@@ -53,33 +210,35 @@ def __init__(
             expiration_secs: Optional[int] = None,
             data_type_name: Optional[str] = None,
             on_error: Any = None,
+            eager: bool = True,
             **kwargs):
         self._vault_property: Optional[str] = vault_property
         self._vault_collection = vault_collection
         self.encryption_type = encryption_type
         self.expiration_secs = expiration_secs
         self._data_type_name = data_type_name
         self.on_error = on_error
+        self.eager = eager
 
         if 'max_length' in kwargs:
             raise ImproperlyConfigured(
                 'max_length is not supported on EncryptedMixin')
 
         super(EncryptedMixin, self).__init__(*args, **kwargs)
 
-    @property
+    @ property
```

Comment:
@ property => @property ?

Reply:
Yes, some autoformatter keeps fucking it up